cdi-in-maven-plugin-helper/src/main/java/org/ndx/aadarchi/maven/cdi/helper/log/MavenLoggingRedirectorHandler.java

```diff
@@ -14,29 +14,42 @@ public MavenLoggingRedirectorHandler(Log log) {
         this.mavenLog = log;
     }
 
+    public static String shortenSource(String originalSource) {
+        String[] parts = originalSource.split("\\.");
+        StringBuilder newSource = new StringBuilder();
+        for (int i = 0; i < parts.length - 1; i++) {
+            if (i > 0) {
+                newSource.append(".");
+            }
+            newSource.append(parts[i].charAt(0));
+        }
+        newSource.append("." + parts[parts.length - 1]);
+        return newSource.toString();
+    }
+
     @Override
     public void publish(LogRecord record) {
-    	Level level = record.getLevel();
+        String source = record.getSourceClassName().length() <= 20 ? record.getSourceClassName() : shortenSource(record.getSourceClassName());
+        Level level = record.getLevel();
 		if(level==Level.OFF)
     		return;
         int levelValue = level.intValue();
 		if (levelValue <= Level.FINEST.intValue()) {
-            mavenLog.debug("(" + record.getSourceClassName() + ") " + record.getMessage(), record.getThrown());
+            mavenLog.debug("<" + source + "> " + record.getMessage(), record.getThrown());
         } else if(levelValue <=Level.FINER.intValue()) {
-            mavenLog.debug("(" + record.getSourceClassName() + ") " + record.getMessage(), record.getThrown());
+            mavenLog.debug("<" + source + "> " + record.getMessage(), record.getThrown());
         } else if(levelValue <=Level.FINE.intValue()) {
-            mavenLog.debug("(" + record.getSourceClassName() + ") " + record.getMessage(), record.getThrown());
+            mavenLog.debug("<" + source + "> " + record.getMessage(), record.getThrown());
         } else if(levelValue <=Level.CONFIG.intValue()) {
-            mavenLog.info("(" + record.getSourceClassName() + ") " + record.getMessage(), record.getThrown());
+            mavenLog.info("<" + source + "> " + record.getMessage(), record.getThrown());
         } else if(levelValue <=Level.INFO.intValue()) {
-            mavenLog.info("(" + record.getSourceClassName() + ") " + record.getMessage(), record.getThrown());
+            mavenLog.info("<" + source + "> " + record.getMessage(), record.getThrown());
         } else if(levelValue <=Level.WARNING.intValue()) {
-            mavenLog.warn("(" + record.getSourceClassName() + ") " + record.getMessage(), record.getThrown());
+            mavenLog.warn("<" + source + "> " + record.getMessage(), record.getThrown());
         } else { //SEVERE
-            mavenLog.error("(" + record.getSourceClassName() + ") " + record.getMessage(), record.getThrown());
+            mavenLog.error("<" + source + "> " + record.getMessage(), record.getThrown());
```

Comment:
Notice we call mavenLog with different log severity in each case, so this could be "made less repeted thanks to a call to a method having a `BiConsumer<String, LogRecord>` as parameter. I'll show you that tomorrow (alongside the unit test)